lib/array.js

```diff
@@ -229,14 +244,17 @@ const find = (
   }
 };
 
-const every = (
-  // Asynchronous every
-  items, // array, incoming
-  fn, // function, (value, callback) => callback(err, fits)
-  //   value - item from items array
-  //   callback - callback function(err, fits)
-  done // function, optional on done callback function(err, result)
-) => {
+// Asynchronous every
+//   items - <Array>, incoming
+//   fn - <Function>,
+//     value - <any>, item from items array
+//     callback - <Function>
+//       err - <Error>
+//       accepted - <boolean>
+//   done - <Function>, on done, optional
+//     err - <Error>
+//     result - <Array>
```

Comment:
This function produces boolean, not an Array:
```suggestion
//     result - <boolean>
```